.docs/content/motion/1.getting-started/1.introduction.md

```diff
@@ -14,8 +14,67 @@ In this quick overview, we'll take a look at some of the APIs that Oku Motion of
 
 It's based on [Motion One](https://motion.dev).
 
+## The `<motion />` component
 
+The core of Motion is the motion component. Think of it as a plain HTML or SVG element, supercharged with animation capabilities.
 
-<!-- 
-- `@oku-ui/motion` all functions
-- `@oku-ui/motion/types` types -->
+```
+<Motion/>
+```
+
+<iframe
+  src="https://stackblitz.com/edit/vitejs-vite-aqkuft?embed=1&file=src%2Ftemplate%2FRefresh.vue&hideExplorer=1&hideNavigation=1&view=preview"
+  style="width:100%; height:500px; border:0; border-radius: 4px; overflow:hidden;"
+></iframe>
+
+## Gestures
+`<motion />` extends Vue's event system with powerful gesture recognisers. It supports hover, press and more.
+
+```html
+<Motion 
+  :hover="{ scale: 1.2 }" 
+  :press="{ scale: 0.8 }" />
+```
+
+<iframe
+  src="https://stackblitz.com/edit/vitejs-vite-qwc8lb?embed=1&file=src%2FDemo.vue&hideExplorer=1&hideNavigation=1&view=preview"
+  style="width:100%; height:500px; border:0; border-radius: 4px; overflow:hidden;"
+></iframe>
+
+## Keyframes
+Set a value as an array and Motion will animate through each of these values in turn.
+
+By default, each keyframe will be spaced evenly throughout the animation, but the exact timing and easing can be configured via the `transition` property.
+
+```html
+<Motion
+    class="box"
+    :animate="{
+      scale: [1, 2, 2, 1, 1],
+      rotate: [0, 0, 180, 180, 0],
+      borderRadius: ['0%', '0%', '50%', '50%', '0%'],
+    }"
+    :transition="{
+      duration: 2,
+      ease: 'easeInOut',
+      times: [0, 0.2, 0.5, 0.8, 1],
+      repeat: Infinity,
+      repeatDelay: 1,
+    }"
+  />
+```
+
+<!-- <iframe
+  src="https://stackblitz.com/edit/vitejs-vite-bqgiwz?embed=1&file=src%2FDemo.vue&hideExplorer=1&hideNavigation=1&view=preview"
+  style="width:100%; height:500px; border:0; border-radius: 4px; overflow:hidden;"
+></iframe> -->
```

Comment:
why comment ?

Reply:
This is a bug of animation.

https://github.com/oku-ui/motion/issues/8

When the bug is fixed,I will add it.